Lib/json/__init__.py

```diff
@@ -357,3 +357,13 @@ def loads(s, *, cls=None, object_hook=None, parse_float=None,
     if parse_constant is not None:
         kw['parse_constant'] = parse_constant
     return cls(**kw).decode(s)
+
+
+def load(filepath, *, cls=None, object_hook=None, parse_float=None,
```

Comment:
This function already exists in the file, look at the function above ``def loads``.
It doesn't take the file as a path, but takes a file descriptor instead.